Q3DC/Q3DC.py

```diff
@@ -369,6 +392,105 @@ def UpdateInterface(self):
                                                     self.fidListComboBoxlineLB.currentNode())
         self.logic.UpdateThreeDView(self.landmarkComboBox.currentText)
 
+    def init_anatomical_legend(self):
+        if self.anatomical_legend is None:
+            for table_node in slicer.mrmlScene.GetNodesByClass('vtkMRMLTableNode'):
+                if table_node.GetAttribute('Q3DC.is_anatomical_legend') == 'True':
+                    self.anatomical_legend = table_node
+            if self.anatomical_legend is None:
+                self.anatomical_legend = slicer.vtkMRMLTableNode()
+                self.anatomical_legend.SetSaveWithScene(False)
+                self.anatomical_Legend.SetLocked(True)
+                slicer.mrmlScene.AddNode(self.anatomical_legend)
+                self.anatomical_legend.SetAttribute('Q3DC.is_anatomical_legend', 'True')
+
+        al = self.anatomical_legend
+        modify = al.StartModify()
+
+        al.RemoveAllColumns()
+        al.AddColumn().SetName('Landmark')
+        al.AddColumn().SetName('Description')
+        al.SetUseColumnNameAsColumnHeader(True)
+
+        al.EndModify(modify)
+
+    def init_anatomical_radio_buttons(self):
+        self.anatomical_radio_buttons = \
+            [qt.QRadioButton(region) for region in self.suggested_landmarks.keys()]
+        for i in range(self.anatomical_radio_buttons_layout.count()-1, -1, -1):
+            self.anatomical_radio_buttons_layout.itemAt[i].widget().setParent(None)
+        for radio_button in self.anatomical_radio_buttons:
+            self.anatomical_radio_buttons_layout.addWidget(radio_button)
+            radio_button.toggled.connect(
```

Comment:
Instead of connecting each radio button, consider adding them to a `QButtonGroup` and connect the button group.
See https://doc.qt.io/qt-5/qradiobutton.html#details

Reply:
Resolving, now tracked in  https://github.com/DCBIA-OrthoLab/Q3DCExtension/issues/36